bindings/experimental/distrdf/python/DistRDF/ComputationGraphGenerator.py

```diff
@@ -10,6 +10,14 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
+import logging
+
+import ROOT
+from .CppWorkflow import CppWorkflow
+
+logger = logging.getLogger(__name__)
```

Comment:
The `logging` format is left to the user in Python. At the time I first implemented this I followed the guide in the [Python docs](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial) that states some good practices. Modules that want to log something should use `logger = logging.getLogger(__name__)` so that whenever the user will write their logger in their application they will at least get something like 
```DistRDF.ComputationGraphGenerator: ...```
Plus any other formatting option. For simplicity, DistRDF offers a predefined logger through `DistRDF.create_logger` so that the user will get something like
```
DEBUG: DistRDF.Proxy[2021-09-08 12:19:42,275]: Created new Count node
```
in their output. It is still worth tracking this discussion in a separate issue, since I believe at some point we want to use ROOT logging system for DistRDF logs right?

Reply:
this a general PyROOT question, but imho we should not mix different logging mechanisms until a decision is taken

Reply:
I totally agree! This logging mechanism is a remnant from PyRDF

Reply:
I agree with having a uniform logging system. I'll leave this logging message as it is now, more as a placeholder than anything else. When we change it for the whole DistRDF we can change this instance too.

Reply:
is there an issue to track this?

Reply:
Yes https://github.com/root-project/root/issues/8982